PYME/recipes/multiview.py

```diff
@@ -91,15 +91,18 @@ def execute(self, namespace):
 
         try:  # try loading shift map as hdf file
             with unifiedIO.local_or_temp_filename(loc) as f:
-                shift_map_source = tabular.HDFSource(f, 'shift_map')
-                shift_map_source.mdh = HDFMDHandler(shift_map_source.h5f)
+                t = tables.open_file(f)
+                shift_map_source = tabular.HDFSource(t, 'shift_map')  # todo - is there a cleaner way to do this?
```

Comment:
This is probably fine - although ensuring that the file gets closed (does `tables.open_file()` return a context manager that you can use with `with`?) on an error would be slightly cleaner. The reference in to the h5 file in HDFSource was removed to avoid dangling references keeping files open (I think this was causing issues with the high throughput analysis).